Doc/faq/windows.rst

```diff
@@ -26,8 +26,8 @@ with running programs from the Windows command line then everything will seem
 obvious; otherwise, you might need a little more guidance.
 
 Unless you use some sort of integrated development environment, you will end up
-*typing* Windows commands into what is variously referred to as a "DOS window"
-or "Command prompt window".  Usually you can create such a window from your
+*typing* Windows commands into what is referred to as a
+"Command prompt window".  Usually you can create such a window from your
```

Comment:
Actually, the [DOS prompt](https://en.wikipedia.org/wiki/dos_prompt) (`COMMAND.COM`) is a _**separate**_ interpreter from the (more) modern Windows Command Prompt (`cmd.exe`) being referred to here in the rest of this section. Both are included on 32-bit versions of Windows to this day (until Windows 11, which dropped the 32-bit edition), but are incompatible, as the former is only for running legacy 16-bit DOS programs and scripts.

Furthermore, at least in my experience with a variety of different user communities in the US and internationally, the term "DOS prompt" to refer to `cmd.exe` is very uncommon at best (whereas "command prompt"/"cmd" is far more widely used, and younger users might not even know what "DOS" is), and it is very unlikely that any user would recognize its meaning but not "Command prompt", its actual name.

Therefore, to avoid outdated, inaccurate and potentially confusing terminology, I elided it here.